### ISSUE TITLE
Pick a las file and choose a log from it

### ISSUE THREAD

**Issue by WesleyTheGeolien** [WesleyTheGeolien/t20-Dash_Well_Viz#31]

**Comment by mharty3** [comment]
Yes. And even better: pick multiple las files, choose a well, and then chose log(s)

**Comment by WesleyTheGeolien** [comment]
@mharty3 I think this is linked to your branch [improve-dash-layout](https://github.com/WesleyTheGeolien/t20-Dash_Well_Viz/tree/improve-dash-layout)?
